Library/Formula/wren.rb

```diff
@@ -0,0 +1,19 @@
+class Wren < Formula
+  homepage "http://munificent.github.io/wren/"
+  url "https://github.com/munificent/wren/archive/efd161cea278e5cc9d6b404451443cf48d4057a8.tar.gz"
```

Comment:
If you need any help with ` head-only ` feel free to shout over there. Misty's comments are more important than mine by the way, considerably so, so if we end up asking similar things please ignore me :smiley_cat:.